src/hooks/navigation/useParticipantProjectsNav.tsx

```diff
@@ -0,0 +1,20 @@
+import { useCallback } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { APP_PATHS } from 'src/constants';
+
+export default function useParticipantProjectsNav(projectId?: number) {
+  const history = useHistory();
+
+  return {
```

Comment:
Nice! Should we have just one navigate hook with all the options exposed? 
`src/hooks/useNavigateTo` and merge this hook with the other.. thoughts?

Reply:
I like it, I'll make that change in here